mlrun/datastore/__init__.py

```diff
@@ -63,6 +63,16 @@
     is_store_uri,
     parse_store_uri,
 )
+from .storeytargets import (
+    CSVStoreyTarget,
+    KafkaStoreyTarget,
+    NoSqlStoreyTarget,
+    ParquetStoreyTarget,
+    RedisNoSqlStoreyTarget,
+    StreamStoreyTarget,
+    TDEngineStoreyTarget,
+    TSDBStoreyTarget,
```

Comment:
As discussed, it looks like these will be visible to the user, and even appear in API docs.